editors/code/package.json

```diff
@@ -1294,6 +1294,11 @@
                 "description": "Style for the self keyword",
                 "superType": "keyword"
             },
+            {
+                "id": "selfTypeKeyword",
+                "description": "Style for the self type keyword",
+                "superType": "keyword"
+            },
```

Comment:
Not entirely sure whether this should be `keyword` or `typeParameter`, but I am leaning towards keyword.